topostats/default_config.yaml

```diff
@@ -38,9 +38,9 @@ grains:
     upper: 1.0
   direction: upper # Options: upper, lower, both (defines whether to look for grains above or below thresholds or both)
   # Thresholding by area
-  smallest_grain_size_nm2: 24 # Size in nm^2 of tiny grians/blobs (noise) to remove.
+  smallest_grain_size_nm2: 50 # Size in nm^2 of tiny grians/blobs (noise) to remove.
   absolute_area_threshold:
-    upper: [ 120, 192 ] # above surface [Low, High] in nm^2 (also takes null)
+    upper: [ 300, 3000 ] # above surface [Low, High] in nm^2 (also takes null)
```

Comment:
Thanks for correcting this, we really need to pin the values in `default_config.yaml` and leave them alone (see #302 where I was hoping to have such discussion on default values, might be another issue too).

If tests need different values the configuration can be updated after having loaded it to dictionary in the test suite.

Reply:
Good point about fixing the values in the test. I do think these values are better defaults though, my mistake for setting the worse ones in the previous PR. 

I don't think that I should be the sole decider on the default value though, I'll try to get some experimentalist input in #302.